docs/questions-and-answers.md

```diff
@@ -0,0 +1,74 @@
+
+### So where does the blockchain come in?
+
+The subtensor blockchain records all the key activity of all the subnets in its ledger. But most importantly, the subtensor blockchain determines the rewards distribution for subnet miners and subnet validators. An algorithm called Yuma Consensus (YC) is always running on the blockchain. Rankings of the subnet miners set by the subnet validators are input to the YC algorithm. Every 12 seconds, the YC algorithm computes the rewards based on these inputs. These rewards (in TAO) are then deposited into the wallets of subnet miners and subnet validators. 
+
+The subtensor blockchain does not mix data from one subnet with another subnet data. It continuously runs the YC algorithm for each subnet separately. Also keep in mind that a subnet does not communicate with another subnet. 
+
+:::tip See also
+See [Introduction](./learn/introduction.md) and [Anatomy of Incentive Mechanism](./learn/anatomy-of-incentive-mechanism.md) next.
+:::
+
+
+## Mining and validation
+
+### I don't understand Bittensor mining and validation
+
+In Bittensor, the term "mining" is not related to Bitcoin mining. Similarly Bittensor "validation" has nothing to do with blockchain validation. In Bittensor, mining is "subnet mining" and validation is "subnet validation". Subnet miners perform some useful task given to them by subnet validators. See the above [General](#general) section. 
+
+:::tip See also
+See more here in [How a subnet works](learn/introduction.md#how-a-subnet-works). 
+:::
+
+### What is the incentive for me to be a miner or a validator, or even create a subnet? 
+
+Your incentive is that you earn TAO. It works like this. Every 12 seconds a new TAO is created (i.e., minted). This single TAO is then distributed among the 32 subnets. So every 12 seconds each subnet gets a fraction of this newly-created TAO, based on the performance of the subnet. This fractional TAO reward that a subnet receives, called emission, is, in turn, distributed within the subnet like this: 
+- 18% of it goes to the subnet owner.
+- 41% goes to subnet validators (this is called dividend).
+- 41% goes to the subnet miners (this is called incentive). 
+
+Like this, every day 7200 TAO (86,400 seconds in a day, one TAO per 12 seconds) are newly created and distributed as rewards. 
+
+:::tip See also
+See [Emissions](./emissions.md) for details, in specific the role of the root network in determining which subnet gets how much reward.
+:::
+
+### I don't want to create a subnet, can I just be a miner or a validator?
+
+Yes. But remember, you have 32 different subnets to choose from. Requirements for mining or validating are subnet-specific. Start with this [Checklist for Validating and Mining](./subnets/checklist-for-validating-mining.md). Then see the step by step instructions in [Register, Validate and Mine](./subnets/register-validate-mine.md).
```

Comment:
Maybe we can link here to the subnet repos?

Lives in the template:
https://github.com/opentensor/bittensor-subnet-template/blob/main/template/subnet_links.py

Reply:
Good point. Right below on line 70 I link to subnet pages doc. Also, in each of the above links, the subnet pages link exists.